storage-initializer/rockcraft.yaml

```diff
@@ -0,0 +1,72 @@
+# Based on https://github.com/kserve/kserve/blob/v0.10.0/python/storage-initializer.Dockerfile
+name: storage-initializer
+summary: Storage initializer for Kserve deployments
+description: "Kserve storage initializer"
+version: "v0.10.0-22.04-1"
+license: Apache-2.0
+base: ubuntu@22.04
+platforms:
+    amd64:
+run-user: _daemon_
+services:
+  storage-initializer:
+    summary: "Kserve storage initializer service"
+    override: replace
+    command: "/storage-initializer/scripts/initializer-entrypoint [ args ]"
```

Comment:
I think we should keep it the same as upstream, because in the [Dockerfile](https://github.com/kserve/kserve/blob/v0.10.0/python/storage-initializer.Dockerfile#L23) the working-dir is set to /work

Reply:
yeah I'm +1 to keeping as is.  Using working-dir makes sense, but having these as close to upstream dockerfiles as we can makes the updates easier